### ISSUE TITLE
Is `textTransform` TextStyle props necessary?

### ISSUE THREAD

**Issue by MoOx** [necolas/react-native-web#905]
**Do you want to request a *feature* or report a *bug*?**

feature

**What is the current behavior?**

`textTransform` is supported in RNW, but does not exist in RN.
**What is the expected behavior?**

`textTransform` is not a "must have" feature and could be probably remove (as we render text with js, we can handle the transformation from here.

**Idea**: what about removing `textTransform`?

Note: I am opening this feature following https://github.com/reasonml-community/bs-react-native/issues/169, feel free to response "I need this feature" and just close this.

**Comment by MoOx** [comment]
Just asking this one property (not all web property). Why this one? Cause it's think it's unnecessary.  That was before thinking of screenreaders (that I use a lot, I "read" a lot via Voiceover on iOS). I never noticed any change. 

I just tried with this gist https://gist.github.com/MoOx/9aba0be2be5135611b8b8a864a504a45 and you are right, there are for some word some differences (but I tried with actual text, not css textTransform). Sorry for the noize :)
